GameLogic/Runtime/GamePreparation.cs

```diff
@@ -36,12 +43,25 @@ private static void cardSwapLists(List<Card> from, List<Card> to, Guid cardGuid)
 
         public void moveCardToPlayerSelection(Guid cardGuid)
         {
+            if (selectionIsFull)
```

Comment:
Instead of empty return u can check if selection is not full at this moment and if so, code in brackets will be evaluated. Otherwise nothing will happen